src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -32,9 +32,12 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 	}
 	const subscriptions = await Subscription.getAllForHost(jiraHost, gitHubAppConfig.gitHubAppId || null);
 
-	// TODO - this should redirect to a you are not configured page instead.
+	// In case if somehow the user lands on this page directly from URL
```

Comment:
nit: can we reword this? It reads a little funny.

Reply:
I rushed with this comment. I've the comments